vendor/k8s.io/client-go/testing/fake.go

```diff
@@ -104,7 +104,10 @@ func (c *Fake) AddReactor(verb, resource string, reaction ReactionFunc) {
 
 // PrependReactor adds a reactor to the beginning of the chain.
 func (c *Fake) PrependReactor(verb, resource string, reaction ReactionFunc) {
+	c.Lock()
```

Comment:
Is this a dependency being updated?

Reply:
No updates, but could be something to go upstream. Was trying to figure out what changes to be made for races, as slice append is not threadsafe, and some reactors get updated at the same time as others as well as logging of what the fake contains which causes a race.

Reply:
We should not carry a patch here - if this is intrinsic to this PR, can you move the lock calls out to the calling code that needs to perform them?  Otherwise we will have a break the next time we rebase.